chart/templates/node.yaml

```diff
@@ -26,7 +26,7 @@ spec:
         - name: node
           image: {{ $registry }}/concordium/{{ .Values.network }}-node:{{ $.Values.image.tag | default $.Chart.AppVersion }}
           command: ["/concordium-node"]
-          {{- include "common.limits" $.resources | nindent 10 }}
+          {{- include "common.limits" $.Values.resources | nindent 10 }}
```

Comment:
What is the guarantee that this will work?

Reply:
Should i try adding a test for regression, or will i just end up testing how the templating engine works?

Reply:
I looked for current tests related to templating, but found none. It makes sense to me to have some tests ala given values file contents that templated outputs reflects those values, but I did not think I should introduce such tests.